gap/Magic.gd

```diff
@@ -323,15 +323,30 @@
 #!
 #!         </List>
 #!     </Item>
-#!     <Mark><A>example_test</A></Mark>
+#!     <Mark><A>extract_examples</A></Mark>
```

Comment:
Ouch, so the manual was actually wrong here... Thanks for catching and fixing this.